src/Window/Context.cs

```diff
@@ -10,7 +9,7 @@ namespace SFML.Window
     /// This class defines a .NET interface to an SFML OpenGL Context
     /// </summary>
     //////////////////////////////////////////////////////////////////
-    public class Context : CriticalFinalizerObject
```

Comment:
If i remember correctly, from my .Net Core Port CriticalFinalizerObject dosn't exist in .Net Core, but it should have been replaced at least by IDisposable see my "Dotnet Core" PR ;)